example/lib/main.dart

```diff
@@ -14,7 +15,119 @@ class MyApp extends StatelessWidget {
       theme: ThemeData(
         primarySwatch: Colors.blue,
       ),
+      // Default example not shows webview until resize the window
       home: WebViewXPage(),
+      // webview_windows sample works without resize
+      //home: WebViewXPageWorkingSample(),
```

Comment:
There shouldn't be two example pages, only one that works for all platforms (that is the purpose of being multi-platform)

Reply:
Yes, I know. It will be deleted. I just put a sample where you can test that how it should be work. As I said, with WebViewX widget the windows webview not load until resize the window. I need help to fix this.

Reply:
Are you sure you read the instructions from Readme.md? You should run the web version with `--web-renderer html` argument until canvaskit is fixed. It works perfectly here, I did not change anything.

Reply:
The issue is not in web version! The issue in this PR with windows WebView. It's work perfectly with included example (that will be deleted), but not appears without resize with WebViewX widget.

Reply:
Oh, I understand. That must be investigated, I cannot tell right now why is this happening.